polar-core/src/vm.rs

```diff
@@ -1117,7 +1117,11 @@ impl PolarVirtualMachine {
         check_errors: bool,
     ) -> PolarResult<QueryEvent> {
         let (field_name, args): (Symbol, Option<Vec<Term>>) = match self.deref(field).value() {
-            Value::Call(Predicate { name, args }) => (
+            Value::Call(Call {
+                name,
+                args,
+                kwargs: None,
```

Comment:
Why are we only matching `None` for kwargs?

Reply:
I think this is related to @samscott89's comment: https://github.com/osohq/oso/pull/405#issuecomment-687264166

Reply:
Correct, it's because there's no support for mixed args in method calls. But there could be.